yarn.lock

```diff
@@ -6493,6 +6495,11 @@ node-fetch@^1.0.1:
     encoding "^0.1.11"
     is-stream "^1.0.1"
 
+node-fetch@^2.3.0:
```

Comment:
now that's ugly :(

node-fetch is for serverside usage only, but bundled on a clientside as well

Reply:
@sunify any ideas how to avoid that?

Options:
1. leave it alone
2. don't use node-fetch, so that fetch-dependent code won't work on a serverside. Affects only `fastSell` helper
3. somehow split the leap-core into browser/node bundles
4. ... ?

Reply:
I don't see node-fetch inside client bundle. Anyway, `node-fetch` package has browser field (https://unpkg.com/node-fetch@2.5.0/browser.js) so bundlers will use it instead of node version